dashboard/config/application.rb

```diff
@@ -80,6 +80,7 @@ class Application < Rails::Application
     cache_bust_path = Rails.root.join('.cache_bust')
     ::CACHE_BUST = File.read(cache_bust_path).strip.gsub('.', '_') rescue ''
 
+    config.assets.paths << Rails.root.join('./public/blockly/js')
```

Comment:
I certainly don't see this causing anything do predownload, but I also suspect I'm not doing this right.  I was hoping you'd know :grin: but I'll do some more reading on this.